### PR TITLE
docs: describe dedicated account dashboard for ent

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 1]
```diff
@@ -109,6 +109,12 @@ session, which is useful for teams that need to collaborate at the terminal.
 
 See [Moderated Sessions](../../access-controls/guides/moderated-sessions.mdx) for more information.
 
+## Dedicated Account Site
```
Comment (ptgott) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:112]:
```suggestion
## Dedicated account site
```

We use sentence case for headings

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 2]
```diff
@@ -109,6 +109,12 @@ session, which is useful for teams that need to collaborate at the terminal.
...
+
+Teleport Enterprise subscriptions include a dedicated account site with their preferred
+subdomain of (teleport.sh)[https://teleport.sh]. The dedicated account site provides
+subscription administrators access to the License file, support links and Enterprise binary downloads.
```
Comment (ptgott) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:116]:
```suggestion
subscription administrators access to the license file, support links and Teleport Enterprise binary downloads.
```
Fixing proper noun usage and including "Teleport" to make this more explicit

**Review by ptgott — APPROVED** [review]

**Review by avatus — APPROVED** [review]

**Comment by roraback** [comment]
One edit: wherever we say "dedicated account site" I would recommend saying "dedicated account dashboard" because we call it a dashboard in account emails and in the UI when it's being spun up.

**Review by roraback — CHANGES_REQUESTED** [review]
One edit: wherever we say "dedicated account site" I would recommend saying "dedicated account dashboard" because we call it a dashboard in account emails and in the UI when it's being spun up.

**Comment by stevenGravy** [comment]
> One edit: wherever we say "dedicated account site" I would recommend saying "dedicated account dashboard" because we call it a dashboard in account emails and in the UI when it's being spun up.

Updated

**Review by roraback — APPROVED** [review]

**Pull request merged**